src/coreclr/dlls/mscorrc/mscorrc.rc

```diff
@@ -495,6 +495,7 @@ BEGIN
 
     IDS_EE_ARRAY_DIMENSIONS_EXCEEDED        "Array dimensions exceeded supported range."
 
+    IDS_EE_OUT_OF_SYNCBLOCKS                "Internal limitation: too many SyncBlocks."
```

Comment:
can the message mention the limit which might indicate a possible leak within the application? Appears that the limit is `2^26`?

Reply:
```suggestion
    IDS_EE_OUT_OF_SYNCBLOCKS                "Internal limitation: attempt to create more than 2^26 SyncBlocks."
```
How about this?

Reply:
looks good.